l10n_es_intrastat_report/models/l10n_es_intrastat_product_declaration.py

```diff
@@ -50,6 +50,9 @@ def _update_computation_line_vals(self, inv_line, line_vals):
         incoterm_id = self._get_incoterm(inv_line)
         if incoterm_id:
             line_vals['incoterm_id'] = incoterm_id.id
+        if self.type == 'dispatches' and self.year >= 2022:
```

Comment:
¿No te da éste error al comparar el año?

`TypeError: '>=' not supported between instances of 'str' and 'int'`

Reply:
En un principio funcionaba correctamente, pero si que es cierto que se debería de transformar. Acabo de subir la corrección para traspasarlos a int.